src/js/cf-expandables.js

```diff
@@ -5,110 +5,168 @@
  * A public domain work of the Consumer Financial Protection Bureau
  */
 
-(function( $ ) {
+( function( $ ) {
+
+  'use strict';
+
+  var _extend = $.extend;
+
+  var Expandable = {
+
+    defaultProperties: {
+      isInAccordion: false,
+      isExpanded:    false,
+      expandedClass: 'expandable__expanded'
+    },
+
+    // Elements that will be cached.
+    ui: {
+      cueOpen:  '.expandable_cue-open',
+      cueClose: '.expandable_cue-close',
+      content:  '.expandable_content',
+      target:   '.expandable_target'
+    },
+
+    create: function create( options ) {
+      var properties = _extend( true, {}, Expandable.defaultProperties, options );
+
+      // Using $.extend to create an instance of the Expandable.
+      // Extending dom elements is generally bad and should be
+      // avoided. `this` is referring to the dom element.
+      _extend( this, Expandable, properties );
+      this.init( );
+    },
+
+    init: function init() {
+      this.isInAccordion = Boolean( this.$el.parents( '.expandable-group' )
+                           .data( 'accordion' ) );
+      this.initUI();
+      this.initEvents();
+    },
+
+    destroy: function destroy() {
```

Comment:
so its just added as a convenience method if you wanted to remove the element from the dom